tembo-cli/src/cmd/apply.rs

```diff
@@ -630,7 +630,16 @@ fn get_extensions(
                             name
                         )));
                     } else {
-                        continue;
+                        let ext_locations = extension_mismatch.unwrap().locations.clone();
+                        if ext_locations.len() > 0 {
+                            if let Some(existing_version) = ext_locations[0].clone().version {
+                                version = existing_version
+                            }
```

Comment:
the else for this block appears to be unhandled condition, meaning in the case where existing_version is None